src/R/Components/Test/MefCompositionTests.cs

```diff
@@ -56,9 +56,5 @@ public class MefCompositionTests : IDisposable {
         public void InteractiveWindowFactoryService() {
             _exportProvider.GetExportedValue<IInteractiveWindowFactoryService>().Should().NotBeNull();
         }
-
-        public void Dispose() {
-            (_exportProvider as IDisposable)?.Dispose();
```

Comment:
We need to figure out then when to dispose it. disposing it here causes core editor to throw since although from our point of view the control is closed, it is still getting some code executed (I guess dispatcher frames age getting pushed). I'll see if I can find safe place to disposed of the container.